src/kubelet/client/connector.go

```diff
@@ -123,7 +148,7 @@ func (dp defaultConnector) getKubeletSchema(kubeletPort int32) string {
 		dp.logger.Debugf("Setting Kubelet Endpoint Schema https since kubeletPort is %d", kubeletPort)
 		return httpsSchema
 	default:
-		dp.logger.Debugf("Schema is unknown since kubeletPort is %d", kubeletPort)
+		dp.logger.Errorf("Cannot automatically figure out schema from non-standard port %d, please set kubelet.schema in the config file.", kubeletPort)
```

Comment:
If we cannot figure out the schema we still go on, so perhaps we could downgrade this to `Info` rather than error.